3.10/installation-linux.md

```diff
@@ -1,38 +1,59 @@
 ---
 layout: default
-description: To install ArangoDB on Linux visit the Download page of the ArangoDB and download the correct package for your Linux distribution. Follow the instructions to use your favorite package.
-title: Installing ArangoDB on Linux 
+description: Download an installation or tar package, or use a package manager
+title: Installing ArangoDB on Linux
 ---
 Installing ArangoDB on Linux
 ============================
 
-To install ArangoDB on Linux:
+You can install ArangoDB on most common Linux distributions. The basic
+installation steps are:
 
-1. Visit the official [Download](https://www.arangodb.com/download){:target="_blank"} page of the
-   ArangoDB web site and download the correct package for your Linux distribution.
-   You can find binary packages for the most common distributions there. Linux Mint: 
-   please use the corresponding Ubuntu or Debian packages. 
-2. Follow the installation instructions on the _Download_ page to use your
-   favorite package manager for the major distributions. After setting up the ArangoDB
-   repository you can easily install ArangoDB using _yum_, _aptitude_, _urpmi_ or _zypper_.
+1. Visit the official [Download](https://www.arangodb.com/download){:target="_blank"}
+   page of the ArangoDB web site.
 
-{% hint 'info' %}
-In addition to installation packages (distribution dependent) a `tar.gz` archive
-is available starting from version 3.4.0.
-{% endhint %}
+2. Click the logo of the distribution that matches your operating system.
+   If you use Linux Mint, click Ubuntu or Debian.
```

Comment:
No such logos at https://www.arangodb.com/download.

Reply:
Right, I had something about CE and EE there but removed it again, can't remember why.